pkg/models/router.go

```diff
@@ -351,3 +351,34 @@ type NetworkRouterBgpNeighborBody struct {
 	DateCreated        string `json:"dateCreated"`
 	LastUpdated        string `json:"lastUpdated"`
 }
+
+type TransportZonesResp struct {
+	NetworkScopes []NetworkScope `json:"networkScopes"`
+	Meta          Meta           `json:"meta"`
+}
+
+type NetworkScope struct {
+	ID            int                `json:"id"`
+	InternalID    string             `json:"internalId"`
+	Visibility    string             `json:"visibility"`
+	DateCreated   string             `json:"dateCreated"`
+	ProviderID    string             `json:"providerId"`
```

Comment:
We may need to make this as computed field. Get the **providerId** from the Transport Zone Data Source. Pass `infra/sites/default/enforcement-points/default/transport-zones/<**providerId**>` as parameter to the Network Creation

Reply:
Thanks for pointing out, added the logic in vmaas repo